.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @stoplightio/team-plaid
+/packages/rulesets/src/asyncapi  <add-asyncapi-team>
```

Comment:
@marbemac could you make sure to create some team and add https://github.com/magicmatatjahu?
@jonaslagoni would you like to be included as well?

Reply:
Sorry for such a delay! Thanks a lot! I feel honored :) If it would be possible @smoya himself was also interested.

Reply:
@P0lip it would be an honor 🙂

Reply:
>  If it would be possible @smoya himself was also interested.

It would be amazing! Very much appreciated if you do @P0lip !

Reply:
The invites have been sent. LMK in case you haven't received one.